src/components/entryTab.tsx

```diff
@@ -110,6 +111,14 @@ export function EntryTab(props: {
     for (const entity of props.entities) {
         const course = entity.getCourse();
         for (const entry of entity.entries) {
+            const hasfinished = entry.hasFinished;
+            if (hasfinished == true){
```

Comment:
細かいですが，`hasFinished`はbool型なので，
```suggestion
            if (hasfinished){
```
にしてもらえると良いかと思います！

Reply:
あ、本当ですね、ありがとうございます！